README.md

```diff
@@ -44,17 +45,20 @@ module.exports = {
 };
 ```
 
-`package.json` tells you about the template. It is necessary state that `"main": "index.js"` so that the reference to the template source in `index.js` is propagated. All templates should contain the keyword `"cordova:template"` so that the template is searchable on npm.
-```javascript
-...
-"main": "index.js",
-...
-"keywords": [
-"cordova:template"
-...
+#### package.json
+This `package.json` holds *information about the template itself* like its name, version etc. All templates should contain the keyword `"cordova:template"` so that the template is searchable on npm. For example:
+```json
+{
+    "name": "cordova-example-template",
+    "version": "1.0.0",
+    "...": "...",
+    "keywords": [
+        "cordova:template"
+    ]
```

Comment:
Can you add `"main" : "index.js",` to this `package.json` example?

Reply:
I intentionally left it out since it is not required. In `cordova-create` we don't look at this property at all but just `require` the module. This resolves to `index.js` even w/out this property.
So this way, there's less noise to make sense of.

Reply:
Sounds good